src/main/java/com/jfrog/ide/idea/configuration/GlobalSettings.java

```diff
@@ -167,19 +186,18 @@ public boolean areArtifactoryCredentialsSet() {
      * The plugin supports reading the JFrog connection details from JFrog CLI's configuration.
      * This allows developers who already have JFrog CLI installed and configured,
      * to have IDEA load the config automatically.
-     *
-     * @return true if connection details from CLI were loaded.
      */
-    public boolean loadConnectionDetailsFromJfrogCli() {
+    public void loadConnectionDetailsFromJfrogCli() {
         try {
             if (serverConfig.readConnectionDetailsFromJfrogCli()) {
                 Logger.getInstance().info("Successfully loaded config connection details from JFrog CLI");
-                return true;
+                MessageBus messageBus = ApplicationManager.getApplication().getMessageBus();
+                messageBus.syncPublisher(ApplicationEvents.ON_CONFIGURATION_DETAILS_CHANGE).update();
```

Comment:
Shouldn't we remove these lines?
Because reloadXrayCredentials sets the credentials from JFrog CLI and returns true